doc/resources.md

```diff
@@ -47,3 +47,7 @@ See [this page](./github.md) for more information.
 - [YouTube](https://www.youtube.com/@NixOS-Foundation) - owner: marketing
 - [Twitter](https://twitter.com/nixos_org) - owner: marketing
 - [Mastodon](https://chaos.social/@nixos_org) - owner: marketing
+<!-- xrefcheck: ignore link -->
```

Comment:
curious what made this link different

Reply:
> curious what made this link different

`< HTTP/2 999` LinkedIn returns status code 999 based on some user agents